packages/firestore/rollup.config.js

```diff
@@ -31,16 +32,21 @@ import pkg from './package.json';
 
 const util = require('./rollup.shared');
 
-// Customize how import.meta.url is polyfilled in cjs nodejs build. We use it to be able to use require() in esm.
-// It only generates the nodejs version of the polyfill, as opposed to the default polyfill which
-// supports both browser and nodejs. The browser support is unnecessary and doesn't work well with Jest. See https://github.com/firebase/firebase-js-sdk/issues/5687
-function importMetaUrlPolyfillPlugin() {
+// Customize how import.meta.url is polyfilled in cjs nodejs build. We use it to
+// be able to use require() in esm. It only generates the nodejs version of the
+// polyfill, as opposed to the default polyfill which supports both browser and
+// nodejs. The browser support is unnecessary and doesn't work well with Jest.
+// See https://github.com/firebase/firebase-js-sdk/issues/5687
```

Comment:
Re-wrapped this comment

Reply:
If you check the link, the ternary was originally removed because the document check was coming up true for Jest which caused it to take the second option which broke Jest somehow, so I think this will break Jest again. I am not sure why Vercel is using the Node bundle in what seems to be a non-Node environment but I'll try to look into that right away.